### DISCUSSION TITLE
Cannot control Crazyflie using Phone

### DISCUSSION THREAD

**Discussion by shrijayan** [bitcraze/discussions#139] [Q&A]
Error : Parameters TOC fetch finished: 0

First of all, we were working with the crazyflie and controlling with an Android Mobile phone suddenly after a restart of this drone the error came.
Please find us a solution for this.

![WhatsApp Image 2022-07-11 at 9 23 35 PM](https://user-images.githubusercontent.com/81805145/178305624-3ceff843-c39f-4b01-8fe9-5b53c6ad5bf4.jpeg)

**Comment by knmcguire** [comment]
Hi!

Some more additional questions:
- Which version of the Crazyflie-firmware does your crazyflie have?
- Which mobile phone type are you using?
- If you connect a crazyradio PA to it your phone with an adapter and used the advanced connection field, do you see the same thing happening?

**Reply by shrijayan** [reply]
Respected @knmcguire,

* We use the Crazyflie-firmware 2022.05
* Android type of Mobile phone
* Sorry we don't have that adapter for the advanced connection field

Yours sincerely,
Shrijayan
https://cpluz.com

**Reply by knmcguire** [reply]
With type of phone, I meant what brand (Samsung, Motorola..), model code  and such and which Android version. 

At least with the latest Android version, we don't see problems with our phones at the office with firmware 2022.05
